src/terrain/clients/metadata/raw.clj

```diff
@@ -162,46 +162,47 @@
 
 (defn list-all-attached-tags
   []
-  (http/get (metadata-url "filesystem" "data" "tags") (get-options)))
+  (:body (http/get (metadata-url "filesystem" "data" "tags") (json-get-options))))
 
 (defn remove-all-attached-tags
   []
-  (http/delete (metadata-url "filesystem" "data" "tags") (delete-options)))
+  (:body (http/delete (metadata-url "filesystem" "data" "tags") (json-get-options))))
 
 (defn list-attached-tags
   [target-id]
-  (http/get (metadata-url "filesystem" "data" target-id "tags") (get-options)))
+  (:body (http/get (metadata-url "filesystem" "data" target-id "tags") (json-get-options))))
 
 (defn update-attached-tags
-  [target-id data-type type body]
-  (http/patch (metadata-url "filesystem" "data" target-id "tags")
-              (post-options body {:data-type data-type
-                                  :type type})))
+  [target-id data-type params body]
+  (:body (http/patch (metadata-url "filesystem" "data" target-id "tags")
+              (json-post-options body
+                                 {:data-type data-type
+                                  :type (:type params)}))))
 
 (defn get-tags-by-value
   [contains limit]
-  (http/get (metadata-url "tags" "suggestions") (get-options (remove-nil-values {:contains contains
-                                                                                 :limit limit}))))
+  (:body (http/get (metadata-url "tags" "suggestions") (json-get-options (remove-nil-values {:contains contains
+                                                                                 :limit limit})))))
```

Comment:
This is really minor, but it might help to line up `:limit` with `:contains` to show that it's part of the same map. I think that IntelliJ has an option to line up the code for you. Otherwise, you can run `lein cljfmt fix src/terrain/clients/metadata/raw.clj` to have it reformat the code for you. That will probably make some additional changes as well. If you prefer, you can also run `lein cljfmt check src/terrain/clients/metadata/raw.clj` to have it display the recommended changes.

Reply:
Ah ok, I'll try that out. I've been lining things up manually because IntelliJ wants to space things out super far (even though it's set to 2 spaces for a tab).